### DISCUSSION TITLE
Don't know how to build task 'shakapacker:install'

### DISCUSSION THREAD

**Discussion by bolpin** [shakacode/shakapacker#289] [General]
I'm following the installation instructions in the README and am seeing this
```
./bin/rails shakapacker:install                                                                                                                            [~] ✭
rails aborted!
Don't know how to build task 'shakapacker:install' (See the list of available tasks with `rails --tasks`)
```

Have tried with an existing Rails6 app during a webpacker -> shakapacker move, as well as with a brand new rails app as described in the README  (rails new myapp --skip-javascript)

**Comment by bolpin** [comment]
It  appears the command has changed from
```
./bin/rails shakapacker:install
```
to 
```
./bin/rails webpacker:install
```

**Comment by justin808** [comment]
This is for version 7 after the names change.

**Comment by ahangarha** [comment]
@bolpin Use the [6-stable](https://github.com/shakacode/shakapacker/tree/6-stable) branch for the documentation of the current release.
